src/osp/Active/SysWire.h

```diff
@@ -287,12 +287,12 @@ class SysWire : public IDynamicSystem
     SysWire(SysWire const& copy) = delete;
     SysWire(SysWire&& move) = delete;
 
-    void update_propigate();
+    void update_propigate(ActiveScene& rScene);
```

Comment:
Since this line is already being modified, might be a good time to fix typo `propigate`->`propagate`